src/components/delivery/Compass.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { selectRoverPosition } from "../../store/telemetrySlice";
+import "./Compass.css";
+
+const Compass = () => {
```

Comment:
There's a lot of extra parenthesis and spaces in this component that makes things a little harder to read. Can you clean it up a little?

For example, there's no need to write `(Math.atan2(...,...))`, you can just remove the outer parenthesis. Same with if statements, no need to do `if ( (Math.abs(...) < ...) )`

Reply:
For Math.atan2(...,...), the parenthesis is need because we are dividing by pi. Other than that, I will fix all the parenthesis.